README.md

```diff
@@ -1,38 +1,40 @@
-# korerorero-mouth-shapes
+# Korerorero / Mouth Shapes
+
+This project is a component of korerorero-reverse-proxy project: <https://github.com/ServiceInnovationLab/korerorero-reverse-proxy>
 
 Exposing DanielSWolf / rhubarb-lip-sync as a service
 
+## .env setup
+
+```bash
+cp .env.example .env
+```
+
```

Comment:
@JacOng17 added some bits about .env
That should make the error go away.

Reply:
:+1: Works now :smile: